### PR TITLE
[WIP] - New formatting for examples + a few new examples

### PR DESCRIPTION
**Pull request by randerzander**
The existing formatting of code snippets includes ">>>" and "..." characters, preventing users from copying and pasting cleanly.

This PR proposes using ".. code-block:: python" markup for examples to make them easier to write and to use.

However, some examples include output. I separated "setup" code lines and the line that produces output with ">>>".

image

But I'm not sure if that's clear enough.. we could also do entirely separate code blocks for output like this:

image

Thoughts?

### CODE REVIEW

**Comment by kkraus14** [comment]
I personally prefer the separate blocks for code and output but welcome other opinions.

File: pygdf/dataframe.py [hunk 1]
```diff
@@ -27,55 +27,76 @@ class DataFrame(object):
...
+    .. code-block:: python
+
+      import pandas as pd
+      from pygdf.dataframe import DataFrame
+      pdf = pd.DataFrame({
+        'a': [0, 1, 2, 3],
+        'b': [0.1, 0.2, None, 0.3]
+      })
+
+      >>> pdf
```
Comment (scopatz) [inline pygdf/dataframe.py]: I don't think it makes any sense to mix blocks like this, especially considering that `python` and `pycon` (Python Console) are different pygments lexers: http://pygments.org/docs/lexers/#pygments.lexers.python.PythonConsoleLexer

**Comment by randerzander** [comment]
Separate code and output blocks it is!

**Comment by randerzander** [comment]
This got way out of date, don't want to try and resolve conflicts. Closing and opening a new PR

**Pull request closed without merging**